### PR TITLE
4.19 release docs

### PR DESCRIPTION
**Pull request by shwstppr**
Changes for 4.19.0.0 changes

----
📚 Documentation preview 📚: https://cloudstack-documentation--381.org.readthedocs.build/en/381/

### CODE REVIEW

**Comment by shwstppr** [comment]
@weizhouapache @nvazquez @DaanHoogland @rohityadavcloud can you please review once?

File: source/releasenotes/about.rst [hunk 1]
```diff
@@ -17,79 +17,52 @@
...
 
-  INSERT INTO cloud.guest_os (uuid, category_id, display_name, created, is_user_defined) VALUES (UUID(), '1', 'CentOS 8', now(), '0');
-  INSERT INTO cloud.guest_os (uuid, category_id, display_name, created, is_user_defined) VALUES (UUID(), '2', 'Debian GNU/Linux 10 (32-bit)', now(), '0');
-  INSERT INTO cloud.guest_os (uuid, category_id, display_name, created, is_user_defined) VALUES (UUID(), '2', 'Debian GNU/Linux 10 (64-bit)', now(), '0');
-  INSERT INTO cloud.guest_os (uuid, category_id, display_name, created, is_user_defined) VALUES (UUID(), '5', 'SUSE Linux Enterprise Server 15 (64-bit)', now(), '0');
-  INSERT INTO cloud.guest_os (uuid, category_id, display_name, created, is_user_defined) VALUES (UUID(), '6', 'Windows Server 2019 (64-bit)', now(), '0');
+Between versions 4.17.x, 4.18.0 and 4.18.1, KVM volume snapshot backups were
+not full snapshots and they rely on the snapshots on the primary storage.
+To prevent any loss of data, care must be taken during revert operation and
+it must be ensured that the source primary storage snapshot file is present
```
Comment (nvazquez) [inline source/releasenotes/about.rst:67]: Perhaps we can add more explicit checks on this part on how to find the file from database?

Comment (shwstppr) [reply source/releasenotes/about.rst:67]: @nvazquez done

**Comment by shwstppr** [comment]
@blueorangutan docbuild

**Comment by rohityadavcloud** [comment]
Overall LGTM two remarks:
Add Oracle Linux as support KVM distro and
Add list of issues fixed ?

File: source/releasenotes/api-changes.rst [hunk 2]
```diff
@@ -13,1912 +13,1362 @@
...
+|                                                  |                                                                                |
+|                                                  | *Changed Parameters:*                                                          |
+|                                                  |                                                                                |
+|                                                  | - ``vpcid`` was 'required' and is now 'optional'                               |
+|                                                  |                                                                                |
++--------------------------------------------------+--------------------------------------------------------------------------------+
 
-Parameters Changed API Commands
--------------------------------
+Default Value Changed for the Parameters of the API commands
```
[the remark this answers is not in the document]
Comment (harikrishna-patnala) [reply source/releasenotes/api-changes.rst:1360]: thanks @shwstppr

**Comment by shwstppr** [comment]
@blueorangutan docbuild

**Pull request merged**